openpathsampling/pathmover.py

```diff
@@ -1503,12 +1502,6 @@ class RandomAllowedChoiceMover(RandomChoiceMover):
     from sub movers that actually can succeed because they have samples in all
     required input_ensembles
 
-    Attributes
-    ----------
-    movers : list of PathMover
-        the PathMovers to choose from
-    weights : list of floats
-        the relative weight of each PathMover (does not need to be normalized)
```

Comment:
I noticed that you removed these inherited attributes from the docstring. Could you double-check that Sphinx is including them when you build the docs, and also that they should up with IPython's `?` trick? I seem to remember at least one of those not respecting docstring inheritance.

Better would be to find a way to get docstrings for inherited attributes to show up in those, though.